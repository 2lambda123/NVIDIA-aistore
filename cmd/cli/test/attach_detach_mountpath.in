ais attach mountpath // FAIL
ais detach mountpath // FAIL
ais attach mountpath something // FAIL "no value for 'something'"
ais detach mountpath something // FAIL "no value for 'something'"
ais attach mountpath $RANDOM_TARGET=/nonexisting // FAIL "fspath "/nonexisting" does not exist, err: no such file or directory"
ais detach mountpath $RANDOM_TARGET=/nonexisting // FAIL "fspath "/nonexisting" does not exist, err: no such file or directory"

ais attach mountpath $RANDOM_TARGET=/tmp/ais
ais detach mountpath $RANDOM_TARGET=/tmp/ais
