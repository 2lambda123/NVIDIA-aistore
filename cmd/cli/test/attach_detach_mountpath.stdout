Node "$RANDOM_TARGET" detached mountpath "$RANDOM_MOUNTPATH"
Node "$RANDOM_TARGET" attached mountpath "$RANDOM_MOUNTPATH"
